### PR TITLE
Upmerge 1.0-rc3 to v1.0

### PR DESCRIPTION
**Pull request by orizohar**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Latest updates from v1.0-rc3 branch (including previous upmerge from 0.11)

### CODE REVIEW

**Comment by orizohar** [comment]
@mukundansundar - please verify IntelliJ recent changes are added correctly
@yaron2 - please verify bindings 
@AaronCrawfis - website related changes and general double checking

**Review by AaronCrawfis — APPROVED** [review]

**Review by yaron2 — APPROVED** [review]

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md [hunk 1]
```diff
@@ -30,12 +30,22 @@ spec:
...
+  - name: saslUsername # Optional.
     value: "user"
-   - name: saslPassword # Optional.
+  - name: saslPassword # Optional.
     value: "password"
+  - name: maxMessageBytes # Optional.
+    value: 1024
 ```
 
+- `topics` is a comma separated string of topics for an input binding.
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md:41]: remove this.

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md [hunk 2]
```diff
@@ -30,12 +30,22 @@ spec:
...
+- `brokers` is a comma separated string of kafka brokers.
+- `consumerGroup` is a kafka consumer group to listen on.
+- `publishTopic` is the topic to publish for an output binding.
+- `authRequired` determines whether to use SASL authentication or not.
+- `saslUsername` is the SASL username for authentication. Only used if `authRequired` is set to - `"true"`.
+- `maxMessageBytes` is the maximum message size allowed for a single Kafka message. Default is 1024.
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md:47]: add this field to the spec metadata fields.

**Review by mukundansundar — COMMENTED** [review]
added some comments ...

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md (hunk 1 above)
Comment (mukundansundar) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md:41]: I will fix it in a follow up PR

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md (hunk 2 above)
Comment (mukundansundar) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/kafka.md:47]: I will fix it in a follow up PR

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**